Sources/Splash/Output/AttributedStringOutputFormat.swift

```diff
@@ -5,19 +5,23 @@
  */
 
 #if os(macOS)
-
 import Cocoa
+#endif
+
+#if os(iOS)
+import UIKit
+#endif
 
 /// Output format to use to generate an NSAttributedString from the
 /// highlighted code. A `Theme` is used to determine what fonts and
 /// colors to use for the various tokens.
 public struct AttributedStringOutputFormat: OutputFormat {
     public var theme: Theme
-
+    
```

Comment:
Can you remove this whitespace? 😇 Pro tip; set Xcode to automatically strip these out using this setting: 
<img width="912" alt="whitespace" src="https://user-images.githubusercontent.com/2466701/44621594-17fa0200-a8a9-11e8-8b59-89b77dd408ed.png">